pesacheck_meedan_bridge/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3.9
+
+WORKDIR /app/pesacheck_meedan_bridge
+COPY . /app/pesacheck_meedan_bridge
+
+RUN pip install virtualenv
+ENV PATH="/app/pesacheck_meedan_bridge/venv/bin:$PATH"
+RUN virtualenv /app/pesacheck_meedan_bridge/venv
+RUN echo "source /app/pesacheck_meedan_bridge/venv/bin/activate" >> ~/.bashrc
+RUN pip install --no-cache-dir -r requirements.txt
+RUN apt-get update && apt-get -y install cron
```

Comment:
Always best practice using `--no-install-recommends` with `apt-get` to avoid installing unnecessary packages.

Question: Shouldn't we do an apt-get update before doing any `pip` installs, though pip ideally reads latest pckgs from pypi